daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -370,78 +369,34 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
+              |                        [infile]
               |
               |Unparse an infoset file, using either a DFDL schema or a saved parser
               |
               |Unparse Options:""".stripMargin)
 
-    descr("unparse a DFDL infoset")
+    descr("Unparse a DFDL infoset")
     helpWidth(76)
 
-    val schema = opt[URI]("schema", argName = "file", descr = "the annotated DFDL schema to use to create the parser.")(fileResourceURIConverter)
-    val rootNS = opt[RefQName]("root", argName = "node", descr = "the root element of the XML file to use.  An optional namespace may be provided. This needs to be one of the top-level elements of the DFDL schema defined with --schema. Requires --schema. If not supplied uses the first element of the schema")
-    val path = opt[String](argName = "path", descr = "path to the node to create parser.")
-    val parser = opt[File](short = 'P', argName = "file", descr = "use a previously saved parser.")
-    val output = opt[String](argName = "file", descr = "write output to file. If not given or is -, output is written to standard output.")
-    val validate: ScallopOption[ValidationMode.Type] = opt[ValidationMode.Type](short = 'V', default = Some(ValidationMode.Off), argName = "mode", descr = "the validation mode. 'on', 'limited', 'off', or a validator plugin name.")
-    val vars = props[String]('D', keyName = "variable", valueName = "value", descr = "variables to be used when unparsing. An optional namespace may be provided.")
-    val tunables = props[String]('T', keyName = "tunable", valueName = "value", descr = "daffodil tunable to be used when parsing.")
-    val config = opt[File](short = 'c', argName = "file", descr = "path to file containing configuration items.")
-    val infosetType = opt[InfosetType.Type](short = 'I', argName = "infoset_type", descr = "infoset type to unparse. Must be one of 'xml', 'scala-xml', 'json', 'jdom', 'w3cdom' or 'sax'.", default = Some(InfosetType.XML))
-    val stream = toggle(noshort = true, default = Some(false), descrYes = "split the input data on the NUL character, and unparse each chuck separately", descrNo = "treat the entire input data as one infoset")
-    val infile = trailArg[String](required = false, descr = "input file to unparse. If not specified, or a value of -, reads from stdin.")
+    val config = opt[File](short = 'c', argName = "file", descr = "Path to XML file containing configuration items")
+    val vars = props[String]('D', keyName = "variable", valueName = "value", descr = "Variables to be used when unparsing. An optional namespace may be provided.")
+    val infosetType = opt[InfosetType.Type](short = 'I', argName = "infoset_type", descr = "Infoset type to unparse. Use 'xml', 'scala-xml', 'json', 'jdom', 'w3cdom', or 'sax'.", default = Some(InfosetType.XML))
+    val output = opt[String](argName = "file", descr = "Output file to write unparsed infoset to. If not given or is -, unparsed infoset is written to stdout.")
```

Comment:
At first I read this it's writing the infoset, and thought something was wrong. I guess the "unparsed infoset" implies that it is writing the the data that is created from unparsing the infoset, and so is technically right, but it stil feels a bit confusing to me. Especially since there is a concept of an "augmented infoset" that is created during unparsing. It's possible someone might interpret this as writing that infoset. I do understand how saying "unparsed data" could also be confusing though. Maybe we just say "Output file to write data to", with the imlication that the data is the data from unparsing the infoset? Maybe I'm overthinking this...

Reply:
Unparsed data makes more sense than Unparsed infoset.